src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -23,7 +25,27 @@
 logger = logging.getLogger(__name__)
 
 
-class ProjectCodeOwnerSerializer(CamelSnakeModelSerializer):
+def validate_association(
+    raw_items: List[Union[UserEmail, ExternalUser, ExternalTeam]],
```

Comment:
Unless your function specifically requires a List, the general rule of thumb is to be less specific for function parameters (`Sequence` in this case) and more specific for return values.